operational/vmware/instance_tag_sync/instance_tag_sync.pt

```diff
@@ -0,0 +1,452 @@
+  resources "rs_cm.credentials"
+end
+
+permission "perm_retrieve_resources" do
+  label "Retrieve Resources"
+  actions   "rs_cm.index","rs_cm.show"
+  resources "rs_cm.instances","rs_cm.volumes","rs_cm.clouds"
+end
+
+permission "perm_tags" do
+  label "Retrieve and add Tags"
+  actions "rs_cm.by_resource","rs_cm.multi_add"
+  resources "rs_cm.tags"
+end
+
+###############################################################################
+# Parameters
+###############################################################################
+
+parameter "param_email" do
+  type "list"
+  label "Email addresses of the recipients you wish to notify"
+end
+
+parameter "param_cloud_name" do
+  type "string"
+  label "Cloud Name"
+end
+
+parameter "param_wstunnel_token" do
+  type "string"
+  label "wstunnel token"
+  no_echo true
+end
+
+parameter "param_vsphere_username" do
+  type "string"
+  label "vsphere username"
+    no_echo true
+end
+
+parameter "param_vsphere_password" do
+  type "string"
+  label "vsphere password"
+  no_echo true
+end
+
+parameter "param_tag_category" do
+  type "string"
+  label "tag category"
+  default "RightScale"
+end
+
+###############################################################################
+# Authentication
+###############################################################################
+
+auth "auth_rs", type: "rightscale"
+
+###############################################################################
+# Resources
+###############################################################################
+resources "clouds", type: "rs_cm.clouds" do
+  filter do
+    cloud_type "vscale"
+    name $param_cloud_name
+  end
+end
+
+resources "instances", type: "rs_cm.instances" do
+  iterate @clouds
+  cloud_href href(iter_item)
+  filter do
+    state ne: ['inactive', 'terminated']
+  end
+  tags '*'
+end
+
+###############################################################################
+# Datasources
+###############################################################################
+
+datasource "ds_instances_and_tags" do
+   iterate @instances
+   field "href",        href(iter_item)
+   field "id",          val(iter_item,'resource_uid')
+   field "name",        val(iter_item,'name')
+   field "state",       val(iter_item,'state')
+   field "tags",        val(iter_item,'tags')
+   field "type",        "instances"
+end
+
+###############################################################################
+# Scripts
+###############################################################################
+
+
+###############################################################################
+# Policy
+###############################################################################
+
+policy "policy_instance_sync" do
+  validate_each $ds_instances_and_tags do
+    summary_template "{{ rs_project_name }} (Account ID: {{ rs_project_id }}): {{ len data }} tags synced on {{ parameters.param_cloud_name }}"
```

Comment:
I am not sure the `summary_template` reads correctly. You are not raising an incident based on the tags to sync, but more on the instances whose tags will be sync'd to VMware right?